server/src/serializers/PastasSerializer.js

```diff
@@ -1,11 +1,28 @@
 class PastasSerializer {
-  static getDetails(pasta) {
+  static async getDetails(pasta) {
     const allowedAttributes = ["id", "name", "description"];
 
     let serializedPasta = {};
     for (const attribute of allowedAttributes) {
       serializedPasta[attribute] = pasta[attribute];
     }
+    const relatedCategory = await pasta.$relatedQuery("category");
+    const relatedReviews = await pasta.$relatedQuery("reviews");
+    serializedPasta.category = relatedCategory;
+    serializedPasta.reviews = relatedReviews;
```

Comment:
On the pasta index page, Im not seeing you use a pasta's category information or info from any of its reviews (you are of course, using that info on the pasta show page)

Once you separate out your serializer methods though this should get resolved